effects/game-of-life/rle2png.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+from sys import argv
+from PIL import Image
+import re
+import argparse
+ 
+header_pattern = re.compile(
+    r'x\s*=\s*(\d+)\s*,\s*y\s*=\s*(\d+)(?:\s*,\s*rule\s*=\s*(\w+\/\w+)\s*)*'
```

Comment:
This is a nightmare to read :(

Could you do it using `str.split` and friends?